package.json

```diff
@@ -6,7 +6,7 @@
     "type": "git",
     "url": "https://github.com/nodaryio/utilities.git"
   },
-  "version": "0.6.1",
+  "version": "0.7.0",
```

Comment:
Revert this. Let's only bump `version` as a step in publishing to avoid forgetting to do it.